app/workers/after_login_job.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class AfterLoginJob
+  include Sidekiq::Worker
+  include Accountable
+
+  sidekiq_options(retry: false)
+
+  def perform(opt)
+    Sentry::TagRainbows.tag
+    user_uuid = opt['user_uuid']
+    return if user_uuid.blank?
+    @current_user = User.find(user_uuid)
+    return if @current_user.blank?
```

Comment:
I really think it will be better to break out these into individual methods. So perform handles each individually.

And each of the individual methods get invoked by perform. Additionally, for maintainability, it might make sense for the hash to include keys that reference those exact methods.